ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/transaction/Transaction.java

```diff
@@ -24,6 +24,11 @@
 import org.web3j.crypto.Sign.SignatureData;
 
 public interface Transaction {
+  BigInteger DEFAULT_GAS_PRICE = BigInteger.ZERO;
+  BigInteger DEFAULT_GAS = BigInteger.valueOf(90000);
+  BigInteger DEFAULT_VALUE = BigInteger.ZERO;
+  String DEFAULT_DATA = "";
+  String DEFAULT_TO = "";
```

Comment:
As the `To` field is optional, shouldn't `null` be the default value?

Reply:
Would make sense, but used "" as that was what being in existing code and also what web3j uses in its Transaction.createContractTransaction method.

Reply:
Right-o